src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -78,6 +84,14 @@ public override void PlatformArrange(Graphics.Rect frame)
 
 			base.PlatformArrange(frame);
 		}
+		
+		void OnPlatformViewAttachedToWindow(object? sender, View.ViewAttachedToWindowEventArgs e)
+		{
+			if (PlatformView is {} platformView && PlatformInterop.GetGlideClearedTag(platformView) && VirtualView is { } virtualView)
```

Comment:
I think this should also have a platformView.IsAlive()... The GC loves to crash things by disposing.